tests/simple/float/f64-cs.wast

```diff
@@ -0,0 +1,153 @@
+;; Test all the f64 operators on major boundary values and all special
```

Comment:
File is named `f64-cs.wast`, should be `f64-c.wast`, right?

Reply:
cs means "conformance simplified", as all hexfloats are converted to floats.

Reply:
Aha, so the pre-processor has been run on these files? Maybe put a README in this folder then that explains it.

Reply:
And please include in the README how to run the preprocessor, with an example.

Reply:
the preprocessor is just the convert.py file, but yes I'll add document somewhere

Reply:
added some words to convert.py. Prefer not adding a README file here because it is more like a temporary solution before we inplement hexfloat.

Reply:
Are we close to having hexfloat working? 

If not, I still prefer some wording somewhere in more prominent documentation. Even if this is meant to be temporary it seems like it could easily become permanent. Having some semi-auto-generated files in the repo that are only distinguished by a naming convention seems to warrant a prominent explanation, especially since the implementer is leaving soon.